charts/seed-controlplane/charts/csi-alicloud/templates/csi-provisioner-dep.yaml

```diff
@@ -0,0 +1,42 @@
+kind: Deployment
+apiVersion: {{ include "deploymentversion" . }}
+metadata:
+  name: csi-provisioner
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kubernetes
+    role: csi-provisioner
+    garden.sapcloud.io/role: controlplane
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubernetes
+      role: csi-provisioner
+  template:
+    metadata:
```

Comment:
secret checksum annotation missing

Reply:
No secret needed for provisioner

Reply:
You are mounting secret "csi-snapshotter"